api/ombpdf/tests/test_management_commands_scrape_memoranda.py

```diff
@@ -94,9 +93,11 @@ def test_scrape_memoranda(monkeypatch):
         lambda p, _: not p.omb_policy_id.endswith('3')
 
     for i in range(1, 5):
-        policy = mommy.make(Policy, omb_policy_id=f'M-0{i}-0{i}')
-        if i == 1:     # Immitate a policy having "already" been processed
-            mommy.make(DocNode, policy=policy)
+        workflow_phase = WorkflowPhases.no_doc.name
+        if i == 1:  # Immitate a policy having "already" been processed
```

Comment:
Not a big deal at all, and I see it's also not new code, but `Immitate` is spelled `Imitate`.